starter-code/js/index.js

```diff
@@ -1,52 +1,199 @@
-function deleteItem(e){
 
-}
-
-function getPriceByProduct(itemNode){
-
-}
-
-function updatePriceByProduct(productPrice, index){
 
+var listOfProduct = [];
```

Comment:
It looks like you're calculating the prices fine, but your web page won't create new items.  Check out the solution code in the Google Drive to compare/contrast